packages/labs/gen-manifest/package.json

```diff
@@ -2,7 +2,7 @@
   "private": true,
```

Comment:
gen-manifest is `private`. Do we expect it to have the patch changes with dependency updates?

Reply:
We can't really help it with our current config of changeset as it auto bumps with dependent update. Looks like it's already had some version bumps too. We'll just have to reset the version when we decide to publish.

Reply:
Agree!